tests/unit/src/kakarot/instructions/test_environmental_information.py

```diff
@@ -38,3 +42,66 @@ async def test_extcodecopy(
             account_registry_address=account_registry.contract_address
         ).call()
         await environmental_information.test__returndatacopy().call()
+
+    @pytest.mark.parametrize(
+        "case",
+        [
+            {
+                "size": 31,
+                "offset": 0,
+                "dest_offset": 0,
+                "id": "size is bytecodelen-1",
```

Comment:
id defined like that will not be used by pytest. See https://docs.pytest.org/en/7.1.x/example/parametrize.html#different-options-for-test-ids